examples/content-source-maps-graphql/app/api/enable-draft/route.ts

```diff
@@ -17,13 +17,13 @@ export async function GET(request: Request) {
   }
 
   // Enable Draft Mode by setting the cookie
-  draftMode().enable();
+  (await draftMode()).enable();
 
   // Override cookie header for draft mode for usage in live-preview
   // https://github.com/vercel/next.js/issues/49927
-  const cookieStore = cookies();
+  const cookieStore = await cookies();
   const cookie = cookieStore.get('__prerender_bypass')!;
-  cookies().set({
+  (await cookies()).set({
```

Comment:
```suggestion
  cookieStore.set({
```

Reply:
Sorry that's the codemod doing that 😅 will change it